plugin/rufo.vim

```diff
@@ -12,13 +16,22 @@ if !exists('g:rufo_errors_buffer_position')
   let g:rufo_errors_buffer_position = 'bottom'
 endif
 
+if get(g:, "rufo_saving", 0)
+  autocmd BufWritePre *.rb silent! call rufo#format_file()
```

Comment:
Could you explain, please, why it used here? Looks like it do the same thing as `rufo#FormatBeforeSave`